make.go

```diff
@@ -1,6 +1,20 @@
 package cursed
 
-// Make makes a []T
-func Make[T any]() []T {
-	return make([]T, len([]T{}))
+import (
+	"strconv"
+	"unsafe"
+)
+
+// Make makes a []T, if an integer is passed in then it make the slice n big
+func Make[T any](n ...int) []T {
+	var cap int
+	switch len(n) {
+	case 0:
+		cap = len([]T{})
+	case 1:
+		cap = **(**int)(unsafe.Pointer(&n))
+	default:
+		panic("invalid amount of lengths passed in: " + strconv.Itoa(len(n)))
```

Comment:
This library should be free of panics (that's what `try` is for!), so maybe this should just set the cap to the number of args passed in! That seems like a definitely reasonable thing to do.
```suggestion
		cap = len(n)
```

Reply:
Smart